src/components/ProgressIndicator/ProgressIndicatorDisplay.js

```diff
@@ -108,17 +110,37 @@ const ProgressIndicatorDisplay = ({
 }) => {
   const [expanded, setExpanded] = useState(false);
 
+  const intl = useIntl();
+  // Wrapper may be a DOM element, which can't handle <FormattedMessage />
+  const ariaIconLabel = intl.formatMessage({
+    description: 'Toggle active step progress indicator icon',
+    defaultMessage: 'Progress step indicator icon',
+  });
+  const toggleStepsTitle = intl.formatMessage({
+    description: 'Active step toggle titles',
+    defaultMessage: 'Show/hide steps',
+  });
+
   return (
     <Card blockClassName="progress-indicator" modifiers={expanded ? [] : ['mobile-collapsed']}>
-      <div
+      {/* Turn a Div with an onClick event into a button for accesibility */}
+      <button
         className={getBEMClassName('progress-indicator__mobile-header')}
+        aria-pressed={expanded ? 'false' : 'true'}
```

Comment:
Reading on [mdn](https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Attributes/aria-pressed):

> Toggle buttons require a full press-and-release cycle to change their value. Pressing and releasing it once changes the value to true. If it's pressed and released again, the value changes back to false.

Pressing the button expands the menu, making it visible, so shouldn't this be swapped?

```suggestion
        aria-pressed={expanded ? 'true' : 'false'}
```

Also following the recommendations, the icon inside the button changes, so that should probably be `aria-hidden="true"` so a screen reader will be reporting the correct thing? Might also need some aria-label to give a more meaningful label to the button instead of the active step title.